samples/sqlservices/src/controllers/mainController.ts

```diff
@@ -701,9 +708,139 @@ export default class MainController implements vscode.Disposable {
+				width: '500px',
+				height: '300px'
+			}).component();
+
+		const lineChart = view.modelBuilder.chart<azdata.LineChartConfiguration>()
+			.withProps({
+				chartId: 'lineChart1',
+				chartType: 'line',
+				configuration: chartExamples.lineConfig,
+				width: '500px',
+				height: '300px'
+			}).component();
+
+		const pieChart = view.modelBuilder.chart<azdata.PieChartConfiguration>()
+			.withProps({
+				chartId: 'pieChart1',
+				chartType: 'pie',
+				configuration: chartExamples.pieConfig,
+				width: '300px',
+				height: '300px'
+			}).component();
+
+		const doughnutChart = view.modelBuilder.chart<azdata.PieChartConfiguration>() // PieChartConfiguration is used for both pie charts and doughnut charts
+			.withProps({
+				chartId: 'doughnutChart1',
+				chartType: 'doughnut',
+				configuration: chartExamples.doughnutConfig,
+				width: '400px',
+				height: '400px'
+			}).component();
+
+		const scatterplot = view.modelBuilder.chart<azdata.ScatterplotConfiguration>()
+			.withProps({
+				chartId: 'scatterplot1',
+				chartType: 'scatter',
+				configuration: chartExamples.scatterConfig,
+				width: '400px',
+				height: '400px'
+			}).component();
+
+		const bubbleChart = view.modelBuilder.chart<azdata.BubbleChartConfiguration>()
+			.withProps({
+				chartId: 'bubbleChart1',
+				chartType: 'bubble',
+				configuration: chartExamples.bubbleConfig,
+				width: '500px',
+				height: '500px'
+			}).component();
 
-	//#region Registrations
+		const polarChart = view.modelBuilder.chart<azdata.PolarAreaChartConfiguration>()
+			.withProps({
+				chartId: 'polarChart1',
+				chartType: 'polarArea',
+				configuration: chartExamples.polarConfig,
+				width: '500px',
+				height: '500px'
+			}).component();
+
+		const radarChart = view.modelBuilder.chart<azdata.RadarChartConfiguration>()
+			.withProps({
+				chartId: 'radarChart1',
+				chartType: 'radar',
+				configuration: chartExamples.radarConfig,
+				width: '500px',
+				height: '500px'
+			}).component();
+
+		const button = view.modelBuilder.button()
+			.withProps({
+				label: 'Click to change bar chart data'
+			}).component();
+
+		button.onDidClick(async () => {
+			// To update data, a new config object must be created and passed.
+			// If the existing one is updated, it's detected as the same object, and "saves" the effort of send propertyChanged events.
+			// This is one way to do that.
+
+			for (let set of chartExamples.barConfig.datasets) {
+				for (let i = 0; i < set.data.length; i++) {
+					set.data[i] = Math.random() * 8;
+				}
+			}
+
+
+			const newConfig: azdata.BarChartConfiguration = {
+				// Spread to pull in existing data/options/properties
+				...chartExamples.barConfig
+
+				// Replace the old data with the new.
+				// Since this example edits the existing configuration object, it's already there (rendering this is second line unnecessary),
+				// but this is how you could insert entirely new data.
+				// datasets: updatedDataSets
+			};
```

Comment:
I don't love having to recreate the entire configuration (and therefore set options when only needing to set new data) instead of just setting the data, so I'm soliciting some feedback for some options:

1. Keep it this way.
2. Separate out data from options ([prototyping in this commit](https://github.com/microsoft/azuredatastudio/commit/b6ed79aad7408fe0fd2e19289da91eaf0b7e74d4)), but TypeScript isn't enforcing the TData-TOption pairings the way I thought it would (meaning someone could pass PieChartOptions with BarChartData).  Not the worst, but I'm looking for solutions to that...
3. Edit data in-place (similar to code sample), then add a `Redraw()` method to the chart component to have it pick up the changes.

Thoughts?  Other ideas?